cmd/completion.go

```diff
@@ -1,28 +1,45 @@
 package cmd
 
 import (
+	"github.com/spf13/cobra"
 	"os"
 
-	"github.com/spf13/cobra"
+	u "github.com/cloudposse/atmos/pkg/utils"
 )
 
-var CompletionCmd = &cobra.Command{
+var completionCmd = &cobra.Command{
 	Use:                   "completion [bash|zsh|fish|powershell]",
-	Short:                 "Generate completion script",
-	Long:                  "To load completions",
+	Short:                 "Generate completion script for Bash, Zsh, fish or PowerShell",
+	Long:                  "This command generates completion scripts for Bash, Zsh, fish and PowerShell",
 	DisableFlagsInUseLine: true,
 	ValidArgs:             []string{"bash", "zsh", "fish", "powershell"},
-	Args:                  cobra.ExactValidArgs(1),
+	Args:                  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
 	Run: func(cmd *cobra.Command, args []string) {
 		switch args[0] {
 		case "bash":
-			cmd.Root().GenBashCompletion(os.Stdout)
+			err := cmd.Root().GenBashCompletion(os.Stdout)
+			if err != nil {
+				u.PrintErrorToStdErrorAndExit(err)
+			}
```

Comment:
These lines appear to be dupe'ed for each case statement. Could we initialize the `var err` above the `var completionCmd` and then turn these lines into just

```suggestion
			err = cmd.Root().GenBashCompletion(os.Stdout)
```

Then further down in the `init()` function we can

```go
func init() {
	err := RootCmd.AddCommand(completionCmd)
	if err != nil {
		u.PrintErrorToStdErrorAndExit(err)
	}
}
```